package.json

```diff
@@ -1,50 +1,52 @@
 {
-    "name": "aion-mining-pool",
-    "version": "0.1.9",
-    "description": "pool",
-    "keywords": [
-        "stratum",
-        "mining",
-        "pool",
-        "server",
-        "poolserver",
-        "aion"
-    ],
-    "license": "GPL-2.0",
-    "author": "ross",
-    "main": "init.js",
-    "bin": {
-        "block-notify": "./scripts/blockNotify.js"
-    },
-    "dependencies": {
-        "async": "*",
-        "bignum": "*",
-        "bindings": "*",
-        "blake2": "file:local_modules/node-blake2",
-        "body-parser": "*",
-        "colors": "*",
-        "compression": "*",
-        "dateformat": "*",
-        "dot": "*",
-        "equihashverify": "file:local_modules/equihashverify",
-        "express": "*",
-        "extend": "*",
-        "moment": "^2.20.1",
-        "mysql": "*",
-        "node-watch": "*",
-        "nonce": "*",
-        "redis": "*",
-        "request": "*",
-        "stratum-pool": "file:local_modules/stratum-pool",
-        "aion-web3": "0.0.3"
-    },
-    "devDependencies": {
-        "gulp": "^3.9.1",
-        "gulp-gzip": "*",
-        "gulp-tar": "*",
-        "moment": "*"
-    },
-    "scripts": {
-        "start": "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PWD/node_modules/stratum-pool/node_modules/equihashverify/build/Release/:$PWD/node_modules/equihashverify/build/Release/ node $NODE_DEBUG_OPTION init.js"
-    }
+  "name": "aion-mining-pool",
```

Comment:
Seems like this PR is 90% code style changes. We should try to separate formatting commits to actual functionality changes(if possible) so we don't go through and read unnecessary code. Also if you have a code style file that you use please share it or create a wiki for it.